app.js

```diff
@@ -12,20 +12,39 @@ const db = pgp({
 // Configure the server and its routes.
 
 const express = require('express');
+const bodyParser = require('body-parser');
+const path = require('path');
+const bcrypt = require('bcrypt');
```

Comment:
I'm assuming that bcrypt is a standard JS encryption library?

Reply:
Yes, I've used it before and from my research it's what most people use.